data/converters/converters.go

```diff
@@ -0,0 +1,194 @@
+var StringNOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeString,
+}
+
+// AnyToOptionalString any value as a string
+var AnyToOptionalString = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableString,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		str := fmt.Sprintf("%+v", v) // the +v adds field names
+		return &str, nil
+	},
+}
+
+// AnyToString any value as a string
+var AnyToString = data.FieldConverter{
+	OutputFieldType: data.FieldTypeString,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		return fmt.Sprintf("%+v", v), nil // the +v adds field names
+	},
+}
+
+// Float64ToOptionalFloat64 optional float value
+var Float64ToOptionalFloat64 = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableFloat64,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		val, ok := v.(float64)
+		if !ok { // or return some default value instead of erroring
+			return nil, fmt.Errorf("[float] expected float64 input but got type %T", v)
+		}
+		return &val, nil
+	},
+}
+
+// Int64ToOptionalInt64 optional int value
+var Int64ToOptionalInt64 = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableInt64,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		val, ok := v.(int64)
+		if !ok { // or return some default value instead of erroring
+			return nil, fmt.Errorf("[int] expected int64 input but got type %T", v)
+		}
+		return &val, nil
+	},
+}
+
+// UInt64ToOptionalUInt64 optional int value
+var UInt64ToOptionalUInt64 = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableUint64,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		val, ok := v.(uint64)
+		if !ok { // or return some default value instead of erroring
+			return nil, fmt.Errorf("[uint] expected uint64 input but got type %T", v)
+		}
+		return &val, nil
+	},
+}
+
+// BoolToOptionalBool optional bool value
+var BoolToOptionalBool = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableBool,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		val, ok := v.(bool)
+		if !ok { // or return some default value instead of erroring
+			return nil, fmt.Errorf("[bool] expected bool input but got type %T", v)
+		}
+		return &val, nil
+	},
+}
+
+// TimeToOptionalTime optional time value
+var TimeToOptionalTime = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableTime,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		val, ok := v.(time.Time)
+		if !ok { // or return some default value instead of erroring
+			return nil, fmt.Errorf("[time] expected time input but got type %T", v)
+		}
+		return &val, nil
+	},
+}
+
+// RFC3339StringToNullableTime .....
```

Comment:
Please leave the comment off until there is an actual comment. Otherwise it tricks the linter, and I have seen this code get into production.